src/config/dynamic-theme-fixes.config

```diff
@@ -1739,6 +1739,19 @@ CSS
 
 ================================
 
+bitwarden.com
+
+INVERT
+img[alt="Github Logo"]
+body > div > section:nth-child(4) > div > div:nth-child(1) > div > blockquote > footer > cite > img
+
+CSS
+.blockquote-header{
+     background-image: url(../images/quote-marks.png) !important;
+}
```

Comment:
```suggestion
IGNORE IMAGE ANALYSIS
blockquote .blockquote-header
```

Reply:
I tried `IGNORE IMAGE ANALYSIS` didn't work for me so i resorted to just CSS.

Reply:
You have to apply it and then give the website a hard refresh.

Reply:
> You have to apply it and then give the website a hard refresh.

I did but didn't work either thats' why i got confused

Reply:
![image](https://user-images.githubusercontent.com/25481501/130271346-d54b18c9-04ba-437d-9321-9c907593d544.png)
does seem to work for me.

Reply:
Ok now i know why:
i was using refrencing only the class in the selector: `.blockquote-header`
but that should work too right?

Reply:
Not really, the current implementation requires the exact selector.

Reply:
> Not really, the current implementation requires the exact selector.

Ok, got it. 
Thanks!.